2024/calcite-react/index.md

```diff
@@ -275,24 +271,12 @@ Functionality)
 > - Finally, as soon as your app is more than a single page, changes in one part
 >   of the app could have unexpected consequences on a different page. Verifying
 >   that everything works correctly in every possibly scenario is very time
->   consuming and error prone. That is why it's an industry best practice to
->   write automated tests for your applications. These could be as simple as
->   making sure your function does some computation correctly, to some as
->   complex as clicking though your app with a virtual mouse, while taking
->   screenshots in the process and comparing them to the expected result.
-
-## Other Sessions Worth Seeing
```

Comment:
given that we are presenting on thursday, by which time most of the sessions are already over, and that we have limited time, I removed this slide in favor of adding this information to the README.md